lib/assets/javascripts/views/orgs/admin_edit.js

```diff
@@ -1,20 +1,25 @@
-$(document).ready(function(){
-  //Validate banner_text area for less than 165 character
-  $("form#edit_org_details").submit(function(){
-      if (getStats('org_banner_text').chars > 165) {
-          alert(__('Please only enter up to 165 characters, you have used') + " " + getStats('org_banner_text').chars + ". " + __('If you are entering an URL try to use something like http://tinyurl.com/ to make it smaller.'));
-          return false;
-      }
+// Import TinyMCE
+import tinymce from 'tinymce/tinymce';
+import ariatiseForm from '../../utils/ariatiseForm';
+import { SHOW_ORG_BANNER_MESSAGE } from '../../constants';
+
+// Returns text statistics for the specified editor by id
+function getStats(id) {
```

Comment:
This is valid, but we've been trying to follow this pattern for function declarations:
`const getStats = (id) => {`
The `function getStats(id)` approach hoists the function to the top of the scope while creating it as an anonymous function attached to a variable keeps it within the current scope